src/app/Unauthenticated.tsx

```diff
@@ -1,7 +1,53 @@
-import Login from 'pages/Login';
+import { Card, CardContent, Grid, Typography } from '@mui/material';
+import { Auth } from '@supabase/ui';
+import Topbar from 'components/header/Topbar';
+import { FormattedMessage } from 'react-intl';
+import { supabase } from 'services/supabase';
+import { getLoginSettings } from 'utils/env-utils';
 
 const Unauthenticated = () => {
-    return <Login />;
+    const loginSettings = getLoginSettings();
+
+    return (
+        <Grid
+            container
+            spacing={0}
+            direction="column"
+            sx={{
+                alignItems: 'center',
+                height: '100vh',
+                justifyContent: 'center',
+            }}
+        >
+            <Topbar isNavigationOpen={false} />
+            <Grid item xs={3}>
+                <Card elevation={24} sx={{ maxWidth: 400, minHeight: 300 }}>
+                    <CardContent>
+                        <Typography
+                            gutterBottom
+                            variant="h6"
+                            component="h2"
+                            sx={{ paddingTop: '1rem', textAlign: 'center' }}
```

Comment:
Interesting to see `rem` used here. Since MUI is innately pixel-centric, I am curious if we should use pixels by default ourselves or adjust MUI's default metric in favor of another.

Reply:
With exception of percentages we should stick with just using the numbers. This is from some older code that I just copied.